src/index.mjs

```diff
@@ -250,6 +288,136 @@ async function translateDocument ({ inputFile, outputFile, targetLangCode, sourc
   }
 }
 
+async function listGlossaries() {
+  try {
+    const [v2Glossaries, v3Glossaries] = await Promise.all([
```

Comment:
Should we consider a scenario in which the API rejects one of these requests but not the other? I can't think of one, but I'm also no longer immersed in GlossaryWorld. As long as we don't, say, start restricting Free users to monolingual glossaries...

Reply:
We only do v3 now so closing this.

Reply:
This makes a lot more sense! Well done